src/hotspot/cpu/x86/nativeInst_x86.cpp

```diff
@@ -313,8 +302,7 @@ void NativeMovRegMem::print() {
 void NativeLoadAddress::verify() {
   // make sure code pattern is actually a mov [reg+offset], reg instruction
   u_char test_byte = *(u_char*)instruction_address();
-  if ( ! ((test_byte == lea_instruction_code)
-          LP64_ONLY(|| (test_byte == mov64_instruction_code) ))) {
+  if (!((test_byte == lea_instruction_code) || (test_byte == mov64_instruction_code))) {
```

Comment:
```suggestion
  if ((test_byte != lea_instruction_code) && (test_byte != mov64_instruction_code)) {
```

Reply:
That looks much nicer. Thanks for the suggestion!